Healthy/Classes/Modules/Dashboard/Views/FoodTagsView/FoodTagsView.swift

```diff
@@ -0,0 +1,102 @@
+import UIKit
+
+final class FoodTagsView: UIView {
```

Comment:
This view need to fix two cases. 
1. We need to handle the initially selected value (Value not index).
2. Let's pass the selection back via closure.